### PR TITLE
Fixed Java Serialization issue

### PR DESCRIPTION
**Pull request by ethankhall**
In #205 we found a regression. This change fixes the issue, and adds a
test so we won't run into it again.

### CODE REVIEW

**Comment by chrisbrake** [comment]
Tested and confirmed this does resolve the issue I reported in #205

File: pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy [hunk 1]
```diff
@@ -97,13 +107,7 @@ class BuildWheelsTask extends DefaultTask implements SupportsWheelCache, Support
     /**
      * Will return true when the package should be excluded from being installed.
```
Comment (sixninetynine) [inline pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy]: is this comment still accurate?

Comment (zvezdan) [reply pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy]: Yes. It still documents what the intended usage of this filter is. It's still a correct documentation for a public field.

File: pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/PythonSerializableTest.groovy [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
```
[the remark this answers is not in the document]
Comment (zvezdan) [reply pygradle-plugin/src/integTest/groovy/com/linkedin/gradle/python/plugin/PythonSerializableTest.groovy]: License checker is rather blunt.
It wants all the files to have the same year.
We didn't spend too much time finding a better one. :-(

**Pull request merged**